dataset/diarization.py

```diff
@@ -0,0 +1,59 @@
+import os
```

Comment:
maybe rename the directory to "datasets" - for the same reason as we use "models": it may be useful to have a variable "dataset" in code, and that leads to name conflict with the module instance